### PR TITLE
[lighthouse] add performance budget docs

### PR DESCRIPTION
**Pull request by kaycebasques**
What's changed, or what was fixed?

- adds official documentation for googlechrome/lighthouse#6053

**Fixes:** N/A

**Target Live Date:** 2019-05-07

- [x] This has been reviewed and approved by @addyosmani or @khempenius 
- [x] I have run `npm test` locally and all tests pass.
- [x] I have added the appropriate `type-something` label.
- [x] I've staged the site and manually verified that my content displays correctly.

**CC:** @petele

### CODE REVIEW

**Comment by addyosmani** [comment]
Thank you so much for getting this content written up, Kayce! A quick sanity-check: are the Lighthouse performance audit docs moving to web.dev for next week or is the canonical source going to continue to be /web?

**Comment by kaycebasques** [comment]
I think that's the long-term plan but wasn't aware of this being a requirement right now. Since the rest of the Lighthouse report UI won't be linking to web.dev next week, as far as I'm aware, I'd suggest keeping it consistent by having it all on /web for now. I might be wrong about all this though so let's get @paulirish & @Meggin in here...

**Comment by robdodson** [comment]
@addyosmani @kaycebasques I think if this is only available from Lighthouse on the command line then it should live on /web. We have audit docs on web.dev but that's because they're relating directly back to the tool running on web.dev/measure.

**Comment by Meggin** [comment]
We are in transition... best to keep tooling on /web... at least for now.

**Comment by addyosmani** [comment]
That sounds reasonable. Thanks for confirming!

**Comment by paulirish** [comment]
This lgtm.

I made a few edits: https://github.com/google/WebFundamentals/pull/7561/commits/91e4dfeb67ce260548f7a4f4f2f5a20e160a175f

File: src/content/en/tools/lighthouse/audits/budgets.md [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+{# wf_updated_on: 2019-05-05 #}
+{# wf_published_on: 2019-04-30 #}
+{# wf_blink_components: Platform>DevTools #}
+
+# Performance Budgets (Keep Request Counts And File Sizes Small) {: .page-title }
+
+## Overview {: #overview }
+
+<aside class="note">
+  The <b>Performance Budgets</b> and <b>Keep Request Counts And File Sizes Small</b> audits both
```
Comment (khempenius) [inline src/content/en/tools/lighthouse/audits/budgets.md]: "Keep Request Counts And File Sizes Small" --> "Keep Request Counts Low And File Sizes Small" here and below

File: src/content/en/tools/lighthouse/audits/budgets.md [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+  link to this page.
+</aside>
+
+After putting in the hard work of improving your site performance, use a *performance budget* to prevent
+your site performance from regressing over time.
+
+The **Keep Request Counts And File Sizes Small** audit lists the total number of requests
+and file size of scripts, images, and so on. This
```
Comment (khempenius) [inline src/content/en/tools/lighthouse/audits/budgets.md]: "file size" --> "transfer size" here and below

File: src/content/en/tools/lighthouse/audits/budgets.md [hunk 3]
```diff
@@ -0,0 +1,283 @@
...
+</figure>
+
+When you define a budget file, the **Performance Budgets** audit reports on the resource types that you've specified.
+But you can still access the information for the rest of the types from the **Keep Request Counts And File Sizes Small**
+audit.
+
+Example JSON output:
+
+```json
+"resource-budget": {
```
Comment (khempenius) [inline src/content/en/tools/lighthouse/audits/budgets.md]: "performance-budget"

File: src/content/en/tools/lighthouse/audits/budgets.md [hunk 4]
```diff
@@ -0,0 +1,283 @@
...
+Follow the instructions below to configure Lighthouse to report when you're over budget. As mentioned
+before, this is only possible when running Lighthouse from the command line. The **Keep Request Counts
+And File Sizes Small** audit is always available in all Lighthouse runtime environments and requires no configuration.
+
+### Define a budget file {: #define }
+
+By convention the budget file is usually called `budget.json` but you can call it whatever you like.
+The example below sets the following budgets:
+
+* 125 kilobytes for all scripts
```
Comment (khempenius) [inline src/content/en/tools/lighthouse/audits/budgets.md]: Maybe reword slightly to indicate that the budget applies to the scripts (& stylesheets) collectively, rather than individually

File: src/content/en/tools/lighthouse/audits/budgets.md [hunk 5]
```diff
@@ -0,0 +1,283 @@
...
+
+<table>
+  <tbody>
+    <tr>
+      <th>Name</th>
+      <th>Type</th>
+      <th>Valid Value(s)</th>
+      <th>Description</th>
+    </tr>
+    <tr>
```
Comment (khempenius) [inline src/content/en/tools/lighthouse/audits/budgets.md]: drop "path" row

File: src/content/en/tools/lighthouse/audits/budgets.md [hunk 6]
```diff
@@ -0,0 +1,283 @@
...
+      <td><code>resourceType</code></td>
+      <td>String</td>
+      <td>
+        <code>total</code>, <code>document</code>, <code>script</code>, <code>stylesheet</code>, <code>image</code>,
+        <code>media</code>, <code>font</code>, <code>other</code>, or <code>third-party</code>.
+      </td>
+      <td>
+        <code>total</code> measures all page resources. <code>document</code> measures HTML document requests.
+        <code>other</code> includes XHR or Fetch requests, and data transfers over WebSocket connections.
```
Comment (khempenius) [inline src/content/en/tools/lighthouse/audits/budgets.md]: "other includes any resource that does not fall into one of the other categories."

File: src/content/en/tools/lighthouse/audits/budgets.md [hunk 7]
```diff
@@ -0,0 +1,283 @@
...
+        <code>third-party</code> measures all resources from third-party domains.
+      </td>
+    </tr>
+    <tr>
+      <td><code>budget</code></td>
+      <td>Integer</td>
+      <td>Any positive integer.</td>
+      <td>The file size limit or resource count limit.</td>
```
Comment (khempenius) [inline src/content/en/tools/lighthouse/audits/budgets.md]: "Depending on context, either the file size limit in kilobytes or the number of requests."

File: src/content/en/tools/lighthouse/audits/budgets.md (hunk 1 above)
Comment (kaycebasques) [reply src/content/en/tools/lighthouse/audits/budgets.md]: Done

File: src/content/en/tools/lighthouse/audits/budgets.md (hunk 2 above)
Comment (kaycebasques) [reply src/content/en/tools/lighthouse/audits/budgets.md]: Done

File: src/content/en/tools/lighthouse/audits/budgets.md (hunk 3 above)
Comment (kaycebasques) [reply src/content/en/tools/lighthouse/audits/budgets.md]: Done

File: src/content/en/tools/lighthouse/audits/budgets.md (hunk 6 above)
Comment (kaycebasques) [reply src/content/en/tools/lighthouse/audits/budgets.md]: Done

File: src/content/en/tools/lighthouse/audits/budgets.md (hunk 7 above)
Comment (kaycebasques) [reply src/content/en/tools/lighthouse/audits/budgets.md]: Done

File: src/content/en/tools/lighthouse/audits/budgets.md (hunk 5 above)
Comment (kaycebasques) [reply src/content/en/tools/lighthouse/audits/budgets.md]: Done

File: src/content/en/tools/lighthouse/audits/budgets.md (hunk 4 above)
Comment (kaycebasques) [reply src/content/en/tools/lighthouse/audits/budgets.md]: Done
